### PR TITLE
@izakp => Hokusai => Master

### PR DESCRIPTION
**Pull request by mzikherman**
Now that DNS is switched, I think this PR is no longer WIP!

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+      - run:
+          name: Deploy
+          command: hokusai staging deploy $CIRCLE_SHA1
+workflows:
+  version: 2
+  default:
+    jobs:
+      - test
+      - push:
```
Comment (mzikherman) [inline .circleci/config.yml]: I'm not 100% sure, but Iit looks like we should replace the `only: hokusai` with `only: master`?

ie- we want to deploy from master.

Maybe we don't need `push:` job?

File: src/index.js [hunk 2]
```diff
@@ -23,11 +24,13 @@ const {
   ENABLE_SCHEMA_STITCHING,
   NODE_ENV,
   QUERY_DEPTH_LIMIT,
-} = process.env
+  SENTRY_PRIVATE_DSN,
```
Comment (mzikherman) [inline src/index.js]: This is already set on the new stacks.

File: src/lib/tracer.js [hunk 3]
```diff
@@ -1,8 +1,16 @@
 import Tracer from "datadog-tracer"
+import config from "config"
 import { forIn, has } from "lodash"
 import { getNamedType, GraphQLObjectType } from "graphql"
 import * as introspectionQuery from "graphql/type/introspection"
 
+const { DD_TRACER_SERVICE_NAME, DD_TRACER_HOSTNAME } = config
```
Comment (mzikherman) [inline src/lib/tracer.js]: We should rename `metaphysics-k8s` to `metaphysics` in this ENV var on the new stacks (although I'm fine leaving it as-is, with the `-k8s`).

Comment (izakp) [reply src/lib/tracer.js]: Will rename once this is merged

File: .circleci/config.yml (hunk 1 above)
Comment (izakp) [reply .circleci/config.yml]: Ah I need to replace `hokusai` with `master` again.  We need the push job to build and push the image

**Pull request merged**